### DISCUSSION TITLE
Disable sound on key release

### DISCUSSION THREAD

**Discussion by MaximMaximS** [GitSquared/edex-ui#1021] [Questions] [answered]
Hello, is there any way to disable the sound of releasing a key on a keyboard? Because it makes sound spam.
Thanks.

**Comment by GitSquared** [comment, marked as answer]
Yes, open the settings (ctrl+shift+s) and toggle on "disableFeedbackAudio".

**Reply by MaximMaximS** [reply]
I already found this, but I would like to disable only the sound on key release and keep sound of a key press.

**Reply by GitSquared** [reply]
Sorry, it's not possible to do that at the moment.

**Reply by MaximMaximS** [reply]
Ok, thanks for the reply :)
